src/main.cc

```diff
@@ -141,6 +141,7 @@ on_shutdown_f(va_list ap)
 	(void) ap;
 	trigger_fiber_run(&box_on_shutdown_trigger_list, NULL,
 			  on_shutdown_trigger_timeout);
+	say_logger_free();
```

Comment:
I'd rather add a new `on_shutdown` trigger to do the task for consistency with other _free methods run from `on_shutdown_f`.

Up to you.

Reply:
I thought about it, but for now it looks like too complex -- we will have only one user here.